src/apps/seelenweg/modules/bar/index.tsx

```diff
@@ -36,24 +36,31 @@ const Separator2: SeparatorWegItem = {
   type: SwItemType.Separator,
 };
 
-function shouldBeHidden(hideMode: HideMode, isActive: boolean, isOverlaped: boolean) {
+function shouldBeHidden(hideMode: HideMode, isActive: boolean, isOverlaped: boolean, associatedViewCounter: number) {

```

Comment:
Why this is needed? The dock is not hidden when focused, so this change does nothing.

Reply:
The associated behaviour from PR desc.

https://github.com/user-attachments/assets/45e5003d-9d9f-4123-b3c2-6757e2b1ac61

Reply:
As pop-up association is not direct in the DOM, this is not direct link and if you leave the associated item, it loses the track of hover, focus is different, but on hover this info must remain to detect user interaction.

Reply:
oh the preview on hover, ok.